packages/scaffolder/src/features/store.spec.ts

```diff
@@ -4,6 +4,7 @@ import { FeatureStore } from './store';
 
 describe('features/store', () => {
   const fixturesPath = path.resolve(__dirname, '../../__tests__/fixtures');
+  const scaffolderFeaturesPackagePath = path.join(__dirname, '..', '..', '..', '..', 'node_modules');
```

Comment:
If you are running the test command from the root of the project you could use `cwd()` instead of finding the path to the root `node_modules`